examples/CMakeLists.txt

```diff
@@ -93,6 +93,9 @@ IF(DEAL_II_COMPONENT_EXAMPLES)
             RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/${DEAL_II_EXECUTABLE_RELDIR}"
             )
           ADD_DEPENDENCIES(examples ${_name}.${_build_lowercase})
+          INSTALL(TARGETS ${_name}.${_build_lowercase}
+            DESTINATION ${DEAL_II_EXECUTABLE_RELDIR}
```

Comment:
But that's not a useful location because none of the input files are located there. I would prefer to install the executable in the `examples/step-XX` directories, alongside their source files. That is, right where they would be if one went to these example directories and compiled things there by hand.